boards/arm/thingy91_nrf52840/thingy91_nrf52840.dts

```diff
@@ -8,8 +8,8 @@
 #include <nordic/nrf52840_qiaa.dtsi>
 
 / {
-	model = "Nordic PCA20035";
-	compatible = "nordic,pca20035", "nordic,nrf52840-qiaa",
+	model = "Nordic THINGY91";
```

Comment:
This is the format used for Thingy:52 upstream now: `model = "Nordic Thingy52 NRF52832";`